app/mailers/application_mailer.rb

```diff
@@ -1,6 +1,7 @@
 # frozen_string_literal: true
 
 class ApplicationMailer < ActionMailer::Base
-  default from: 'from@example.com'
+  # default from: 'from@example.com'
```

Comment:
how do we define the from ?

Reply:
@mod Please, go to /config/initializers/devise.rb line 22